inferentia/scripts/gen_triton_model.py

```diff
@@ -272,9 +272,8 @@ def get_tensorflow_initialize_impl():
 
         # TODO: Validate input/output from the model
 
-        # TODO: NEURONCORE_GROUP_SIZES is deprecated by AWS Neuron
         group_sizes = [str(1)] * cores_per_instance
```

Comment:
group_sizes -> visible_cores.

The concept of group_sizes and visible_cores are fundamentally different.
NEURON_RT_VISIBLE_CORE is defined as "Range of specific NeuronCores needed by the process"

Reply:
I see. Tensorflow processing group size, while pytorch is processing range. Thanks for catching this. Updated by dropping the "switch to NEURON_RT_VISIBLE_CORE" commit.

Reply:
There are also some changes in how `NEURONCORE_GROUP_SIZES` vs `NEURON_RT_NUM_CORES` is set, the previous one is `[1, 1, 1, 1]` and the current one is `4`. This force push makes this change.

Reply:
Thanks!